src/lib/device_tree/src/lib.rs

```diff
@@ -9,11 +26,25 @@ use core::fmt;
 use model::{Driver, Result};
 use wrappers::SectionReader;
 
+/// Special value that every device tree in FDT format starts with.
 pub const MAGIC: u32 = 0xd00dfeed;
+// TODO: Explain why max depth is 16.
```

Comment:
This is just an arbitrary number. The largest nesting level of device tree supported. With a fixed max, we can guarantee the stack won't overflow.

Reply:
Thanks for explaining! I updated the comment.